modules/org.restlet.ext.apispark/src/org/restlet/ext/apispark/internal/model/Operation.java

```diff
@@ -35,13 +35,13 @@
 public class Operation {
 
     /** Mediatypes consumed by this operation */
-    private List<String> consumes;
+    private List<String> consumes = new ArrayList<>();
```

Comment:
I'd better fix the issue completely once. 
I'll add a method to add elements to the list as in `Representation#sections`.
The method should initialize the list if it is null.